python/lsst/afw/geom/utils.py

```diff
@@ -262,16 +331,24 @@ def _compareWcsOverBBox(wcs0, wcs1, bbox, maxDiffSky=0.01*arcseconds,
 
 def wcsAlmostEqualOverBBox(wcs0, wcs1, bbox, maxDiffSky=0.01*arcseconds,
                            maxDiffPix=0.01, nx=5, ny=5):
-    """!Return True if two WCS are almost equal over a grid of pixel positions, else False
-
-    @param[in] wcs0  WCS 0 (an lsst.afw.geom.SkyWcs)
-    @param[in] wcs1  WCS 1 (an lsst.afw.geom.SkyWcs)
-    @param[in] bbox  boundaries of pixel grid over which to compare the WCSs (an lsst.afw.geom.Box2I or Box2D)
-    @param[in] maxDiffSky  maximum separation between sky positions computed using Wcs.pixelToSky
-        (an lsst.afw.geom.Angle)
-    @param[in] maxDiffPix  maximum separation between pixel positions computed using Wcs.skyToPixel
-    @param[in] nx  number of points in x for the grid of pixel positions
-    @param[in] ny  number of points in y for the grid of pixel positions
+    """Return True if two WCS are almost equal over a grid of pixel positions, else False
```

Comment:
Though it's a bit more content, the standard pattern is:

```rst
Test if two WCS are almost equal over a grid of pixel positions.

[...]

Returns
-------
almostEqual : `bool`
    `True` if the WCS are most equal or `False` otherwise.
```